Filtered_kernel/include/CGAL/Lazy_kernel.h

```diff
@@ -307,6 +308,162 @@ class Lazy_kernel_base
   // typedef void Compute_z_3; // to detect where .z() is called
   // typedef void Construct_point_3; // to detect where the ctor is called
 
+  struct Construct_point_2 : public BaseClass::Construct_point_2
+  {
+#ifndef CGAL_CFG_MATCHING_BUG_6
+    using BaseClass::Construct_point_2::operator();
+#else // CGAL_CFG_MATCHING_BUG_6
+
+#ifdef CGAL_CFG_NO_CPP0X_VARIADIC_TEMPLATES
```

Comment:
MSVC 2012 is the last MSVC version without variadic templates? If we drop its support, we can remove that big piece of code.